tools/mo/unit_tests/mo/bom_test.py

```diff
@@ -7,16 +7,19 @@
 import unittest
 from itertools import islice
 
+from openvino.tools.mo.utils.utils import get_mo_root_dir
+
 dir_patterns_to_skip = ['.*__pycache__.*']
 file_patterns_to_skip = ['.*\\.DS_Store$',
                          '.*\\.swp',
-                         '.*\\.pyc$']
+                         '.*\\.pyc$',
+                         'requirements.*\.txt',
+                         '.*version.txt']
```

Comment:
I guess it can be just `'version.txt'`

Reply:
yeah, `version.txt` also should be enough. 
But can we fix that separately? With bunch of other changes it they will be requested

Reply:
done